SPECS/kernel/config_aarch64

```diff
@@ -527,7 +527,6 @@ CONFIG_ARM64_EPAN=y
 # end of ARMv8.7 architectural features
 
 CONFIG_ARM64_SVE=y
-CONFIG_ARM64_SME=y
```

Comment:
Result of microsoft/CBL-Mariner-Linux-Kernel@22aaaa7

ARM Scalable Matrix Extension (SME) is being dubbed "BROKEN" for now due to many issues including the corruption of the FPSIMD/SVE/SME state of arbitrary tasks.

More on ARM SME can be read at
KConfig
linux kernel upstream documentation
ARM blog